### ISSUE TITLE
Vivaldi Browser Hovering Nuttiness

### ISSUE THREAD

**Issue by earl-gresh** [koekeishiya/yabai#1635]
When hovering over certain links or buttons using the Vivaldi browser, it seems to drive Yabai nuts in BSP mode. I think what's happening is Yabai is trying to create a new window for the pop up text that is displayed. See attached image when in float mode.

![image](https://user-images.githubusercontent.com/114691342/219786244-9dceaba1-fd3e-4438-b66d-2c9f09cc1df3.png)

Otherwise, in BSP mode, here's an example of what it looks like on hover:

![2023-02-17_12-22-04 (1)](https://user-images.githubusercontent.com/114691342/219786371-4bcff842-d40d-463f-8815-63dc4e2c4633.gif)

Other than running Vivaldi in "float" mode or using another browser, does anyone have a fix for this?

**Comment by cabbage** [comment]
I've set up some custom debug logs for myself and saw that indeed Vivaldi is opening a new window for those tooltips. 

``` bash
# debugging
log_window="yabai --message query --windows --window \$YABAI_WINDOW_ID | jq -r '\"\(.id),\\\"\(.app)\\\",\\\"\(.title)\\\",is-floating:\(.\"is-floating\")\"' >> ~/tmp/yabai_win_titles.log"
yabai --message signal --add event='window_created' app='^Vivaldi$' action="${log_window}"
yabai --message signal --add event='window_destroyed' app='^Vivaldi$' action="${log_window}"
yabai --message signal --add event='window_title_changed' app='^Vivaldi$' action="${log_window}"
```

I don't get this issue as I've a rule that turns off management for all windows which are not the main window:
```bash
yabai --message rule --add app='^Vivaldi$' title='^.+ - Vivaldi$' manage=on # idk anymore why I did this. seems redundant
yabai --message rule --add app='^Vivaldi$' title!='^.+ - Vivaldi$' manage=off
```

**Comment by earl-gresh** [comment]
Thanks so much for your help! It gave me some ideas on troubleshooting Yabai that I didn't know about.

In the end I was able to just add a single rule:
yabai --message rule --add app='^Vivaldi$' title='' manage=off

Also, I noticed that when Yabai is the only App on a space, the pop-up text is displayed on hover correctly. But when I have another app sharing the same space as Vivaldi (in both BSP and Float), there's no popup text displayed. But hey, I can live with that ;-)

Thanks again, I'm stoked I can go back to using Vivaldi as my primary browser.
